src/components/SecondaryButton.jsx

```diff
@@ -0,0 +1,28 @@
+import styled from "@emotion/styled";
+
+const StyledButton = styled.button`
+  background-color: ${(props) => props.theme.secondaryButton.bgColor};
+  border: none;
+  color: ${(props) => props.theme.colors.violet};
+  padding: ${(props) => props.theme.layout.space200}
+    ${(props) => props.theme.layout.space500};
+  text-align: center;
+  text-decoration: none;
+  font-size: 16px;
+  cursor: pointer;
+  border-radius: 5px;
+
+  &:hover {
```

Comment:
Can you include the `focus` state here as well? Please don't forget about the `PrimaryButton` 😄